reference/ft_analysispipeline.md

```diff
@@ -12,15 +12,15 @@ tags:
  Use as
    output = ft_analysispipeline(cfg, data)
 
- The first cfg input contains the settings that apply to the behaviour of this
+ The first cfg input contains the settings that apply to the behavior of this
```

Comment:
this will not be persistent. The reference documentation is automatically generated from the matlab files. So the help section in the matlab code should be updated. This change does not hurt, so I'll leave it in for now.

Reply:
https://github.com/fieldtrip/fieldtrip/pull/1046